src/pluginmanager.cpp

```diff
@@ -5308,6 +5309,7 @@ void PlugInManager::OnEndPerformCurlDownload(wxCurlEndPerformEvent &ev)
     
     if( m_pCurlThread )
     {
+		m_pCurlThread->Wait();
```

Comment:
It is the stupid VS 12 editor. I type spaces and it seems to replace with tabs.

Best regards,
Dan Dickey

> On Oct 13, 2015, at 6:14 AM, Jan Pieter Waagmeester <notifications@github.com> wrote:
> 
> In src/pluginmanager.cpp:
> 
> > @@ -5308,6 +5309,7 @@ void PlugInManager::OnEndPerformCurlDownload(wxCurlEndPerformEvent &ev)
> >      
> >      if( m_pCurlThread )
> >      {
> > +		m_pCurlThread->Wait();
> you have tabs instead of spaces here...
> 
> —
> Reply to this email directly or view it on GitHub.
>